gtsfm/utils/io.py

```diff
@@ -12,6 +12,8 @@
 from PIL import Image as PILImage
 from PIL.ExifTags import GPSTAGS, TAGS
 
+import gtsfm.utils.reprojection as reproj_utils
+import gtsfm.utils.images as image_utils
```

Comment:
this import should be placed above, following an alphabetical ordering.

Reply:
thanks, i'll fix it

Reply:
fixed